src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -19,7 +32,18 @@
      *                 В любом другом случае бросить DateTimeException
      **/
     public String todayDate(DateTimePart datePart) {
-        return "Today";
+        String result = datePart == DateTimePart.FULL ? TODAY_DATE.toString()
+                : (datePart == DateTimePart.DAY
+                ? TODAY_DATE.format(DateTimeFormatter.ofPattern("dd"))
```

Comment:
```suggestion
                ? TODAY_DATE. getDayOfMonth()
```
аналогічно можна використати стандартні методи для решти випадків